Sources/VSM/StateContainer/StateContainer+Binding.swift

```diff
@@ -14,12 +14,7 @@ import SwiftUI
 
 public extension StateContainer {
     
-    /// Creates a unidirectional, auto-observing `Binding<Value>` for the `State` using a `KeyPath` and a basic closure.
-    /// **Not intended for use when`State` is an enum.**
-    /// - Parameters:
-    ///   - stateKeyPath: `KeyPath` for a `Value` of the `State`
-    ///   - observedSetter: Converts the new `Value` to a new `State`, which is automatically observed
-    /// - Returns: A `Binding<Value>` for use in SwiftUI controls
+    // See StateBinding for details
```

Comment:
You should provide a DocC link to `StateBinding` from here. Improves the ergonomics of using the popup help. (This applies to a bunch of places below here).

Reply:
I probably should have left a PR comment explaining this because it's very nuanced around how Xcode and DocC works.

The popup help and generated docs infer the comments from the protocol, so we're good on that front. In addition, DocC links don't work in `//` comments. If I use a `///` comment, then it overrides the automatic documentation that this function gets from its protocol conformance.